src/atoms/ErrorMessage/error_message.module.scss

```diff
@@ -8,11 +8,11 @@
 }
 
 .warning {
-  color: color('secondary-2');
+  color: color('secondary-1');
 }
 
 .error {
-  color: color('primary-1');
+  color: color('utility-red');
```

Comment:
Under [the new design for invalid inputs](https://www.figma.com/file/iVP8PicuOCAkB7NpswyAWK/Mortar---RCL-color-updates?node-id=2%3A699&viewport=-852%2C-494%2C1.277634859085083), the border and text color should be red to distinguish from the primary brand color.

See also: `src/molecules/formfields/shared/formfields.module.scss`

Reply:
i always found it odd that we associated our _main_ brand color with an error state